vmdb/Gemfile

```diff
@@ -11,9 +11,13 @@ gem "jquery-rjs", "=0.1.1", :git => 'https://github.com/amatsuda/jquery-rjs.git'
 gem 'angularjs-rails', '=1.2.4'
 gem 'jquery-rails', "=2.1.4"
 
+gem "win32-process", "~>0.8.0", :require => false, :platforms => [:mswin, :mingw]
```

Comment:
Yeah, @djberg96 bundler has solved the singular platform well but has always stayed away from adding multiple platform support, probably because it's hard.

I like your win_platform? idea. 

Thoughts @Fryguy ?